packages/manager/modules/pci/src/projects/project/storages/databases/add/translations/Messages_fr_FR.json

```diff
@@ -27,6 +27,8 @@
   "pci_database_add_subnet_add": "Créer un sous-réseau",
   "pci_database_add_subnet_loading": "Chargement de sous-réseaux",
   "pci_database_add_step6_title": "Configuration du service de base de données",
+  "pci_database_add_command_description": "La commande suivante vous permet de créer votre service grâce à",
+  "pci_database_add_command_api_link": "notre API",
```

Comment:
This sentence should not be splitted for translation matters. Maybe the whole text should be "linkable"?

Reply:
Hi @JeremyDec 
As explained in https://github.com/ovh/manager/pull/5863#discussion_r763420786, we splitted this sentence into two different sentences with the second one being linkable. Please teel me if there is still an issue.